src/libexpr/eval.hh

```diff
@@ -187,10 +187,35 @@ public:
     Value vEmptyList;
 
     /**
-     * Null constant.
+     * `null` constant.
+     *
+     * This is _not_ a singleton. Pointer equality is _not_ sufficient.
      */
     Value vNull;
 
+    /**
+     * `true` constant.
+     *
+     * This is _not_ a singleton. Pointer equality is _not_ sufficient.
+     */
+    Value vTrue;
+
+    /**
+     * `true` constant.
+     *
+     * This is _not_ a singleton. Pointer equality is _not_ sufficient.
+     */
+    Value vFalse;
+
+    /** `"regular"` */
+    Value vStringRegular;
+    /** `"directory"` */
+    Value vStringDirectory;
+    /** `"symlink"` */
+    Value vStringSymlink;
+    /** `"unknown"` */
+    Value vStringUnknown;
```

Comment:
Maybe make the doc refer to the builtin for which this is for?

Reply:
That info could bitrot, and is easy enough to find reliably. It should take about 1 second in your IDE/editor to find the usages or occurrences.

Reply:
Guess I need to improve my editor setup :)

Reply:
In vscode, either:
- Shift+F12 for references (needs language server)
- Double click and Ctrl+Shift+F for dumb grep

I'm sure something similar is available or achievable elsewhere.